manifests/m/Mozilla/Firefox/Beta/95.0b9/Mozilla.Firefox.Beta.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=NVS1.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Mozilla.Firefox.Beta
+PackageVersion: 95.0b9
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
```

Comment:
```suggestion
```
These are the InstallerSwitches for the exe, we should also add support for the x64/x86 exe and the x86 msi

I can fix this pretty quick if you want

Reply:
I read in a Firefox support article to add the switches, that's why I added them and it only supports msi installers.

Reply:
Yeeah that article seems a bit drunk, not sure why they say /S for a MSI since thats not a valid argument, they also keep talking about the "full installer" which is the EXE, which does indeed need /S

Reply:
OK I am closing the pull request